integration/test-network/setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ -z "${FPC_PATH}" ]]; then
+  echo "Error: FPC_PATH not set"
+  exit 1
+fi
+
+backup() {
+  FILE=$1
+  BACKUP="${FILE}.backup"
+
+  if [[ -e "${BACKUP}" ]]; then
+    cp "${BACKUP}" "${FILE}"
+  else
+    cp "${FILE}" "${BACKUP}"
+  fi
+}
+
+FABRIC_SAMPLES=${FPC_PATH}/integration/test-network/fabric-samples/
+
+#cd ${FABRIC_SAMPLES} && curl -sSL https://bit.ly/2ysbOFE | bash -s -- -s
+
+CORE_PATH=${FABRIC_SAMPLES}/config/core.yaml
+DOCKER_PATH=${FABRIC_SAMPLES}/test-network/docker/docker-compose-test-net.yaml
+
+# TODO the current `setup.sh` has an issue that it is not idempotent
+# In particular, when starting nodes using the ./network up ... script won't work if
+# docker-compose-test-net.yaml contains redundant volume entries!
+# maybe create a backup copy of `core.yaml` and `docker-compose-test-net.yaml` first and always work from there.
+backup ${CORE_PATH}
+
+yq m -i -a=append ${CORE_PATH} core_ext.yaml
+
+peers=("peer0.org1.example.com" "peer0.org2.example.com")
+
+backup ${DOCKER_PATH}
+
+# Also there is another issue with this approach here. When working completely inside the FPC dev-container,
+# the volume mounts won't work. The reason is that the docker daemon provided by the host cannot parse volume paths.
+# For instance, ${FPC_PATH} inside dev-container is `/project/src/github.com/hyperledger-labs/fabric-private-chaincode/`.
+# The correct path from the docker daemon perspective is something like
+# `/Users/marcusbrandenburger/Developer/gocode/src/github.com/hyperledger-labs/fabric-private-chaincode/` (in my case on the mac).
+
+# Using DOCKERED_FPC_PATH fixes our custom volumes but still there is an issue default mounts in `docker-compose-test-net.yaml`
+# TODO need to be fixed
+
+if [[ -n "${DOCKERD_FPC_PATH}" ]]; then
```

Comment:
```suggestion
if [ ! -z ${DOCKERD_FPC_PATH+x} ]; then
```
so it works also if var is not defined